conf/cassandra.yaml

```diff
@@ -2614,3 +2621,124 @@ drop_compact_storage_enabled: false
 #   compatibility mode would no longer toggle behaviors as when it was running in the UPGRADING mode.
 #
 storage_compatibility_mode: NONE
+
+# At least 20% of disk must be unused to run incremental repair. It is useful to avoid disks filling up during
+# incremental repair as anti-compaction during incremental repair may contribute to additional space temporarily.
+# if you want to disable this feature (the recommendation is not to, but if you want to disable it for whatever reason)
```

Comment:
I think like `reject_repair_compaction_threshold`, we should default this to not do anything as to not change the behavior (should not reject repairs by default, users should have to opt into it).  Was thinking and I realized that there can be a possible downside of this feature, which is if you disable IR while you are low on disk, you might not be able to compact away shadowed data or tombstones because data exists in both the repaired/unrepaired set and can't be compacted together.

Reply:
Just adding another comment here in case this was missed.  I think it may be reasonable to keep this default behavior, but we should just make note of this somewhere in the NEWS entry or upgrade guide.